src/test/SmartComponents/redux/reducers.test.js

```diff
@@ -3,7 +3,7 @@ import types from '../../../SmartComponents/redux/types';
 
 const compose = {
     'xxxx-xxxx-xxxx-xxxx': {
-        state: 'uploading',
+        status: 'uploading',
```

Comment:
See also `redux/actions.test.js`

Reply:
Nice,  I also exchanged the key with a dummy uuid.